python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -2091,20 +2089,19 @@ def __setattr__(self, attr, value):
 
             # reduce duplicate values here, leave only duplicates for LCG,
             # where we can have replicas
-            uniqueValuesDict = []
-            uniqueValues = []
+            uniqueValuesDict = oDict()
```

Comment:
I've mixed feelings about this given that I've seen some bad user code which expects the order here to be preserved. With this in mind a think an ordered dict is sensible unless we want to start educating users about this when their scripts break?